Fix grammatical error in the README.rst (#16587)
matrix-org/synapse: README.rst

in earnest today.

Briefly, Matrix is an open standard for communications on the internet, supporting

> federation, encryption, and VoIP. Matrix.org has more to say about the `goals of the

Reviewer: Nit: the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) is a point of style, not grammar. This is true of a few other changes too.

Author: so should I remove the comma or let it be there

Reviewer: I don't mind either way, I'm just pointing out that this doesn't match the PR title.

Synapse has a variety of `config options
<https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html>`_

> which can be used to customise its behavior after installation.

Reviewer: Synapse's authors are largely British, but I don't think we are committed to British English. Though again, I wouldn't consider that grammar (contrary to the PR title).

Reviewer 2: We use British English in all our documentation. 😄 (At least when you keep me away from it.)

(pronounced "at localpart on my dot domain dot name").

> When logging in, you will need to specify a "Custom server".  Specify your

Reviewer: This changes the meaning. The point of "as when" is to emphasise that the upcoming details apply to another scenario too.

```suggestion
As when logging in, you will need to specify a "Custom server". Specify your
```

**They are not where accounts or credentials are stored - these live on home
servers. Identity Servers are just for mapping 3rd party IDs to matrix IDs.**

This process is very security-sensitive, as there is an obvious risk of spam if it

> is too easy to sign up for Matrix accounts or harvest 3PID data. In the longer

Reviewer: The suggestion "the longer the term" is not correct.

Additionally, longer term is an adjective; this reads better with a noun in its place. I suggest we hyphenate it.

```suggestion
is too easy to sign up for Matrix accounts or harvest 3PID data. In the long-term,
```

> the term, we hope to create a decentralised system to manage it (`matrix-doc #712

Reviewer: ```suggestion
we hope to create a decentralised system to manage it (`matrix-doc #712
```

<https://github.com/matrix-org/matrix-doc/issues/712>`_), but in the meantime,
the role of managing trusted identity in the Matrix ecosystem is farmed out to
a cluster of known trusted ecosystem partners, who run 'Matrix Identity
Servers' such as `Sydent <https://github.com/matrix-org/sydent>`_, whose role
is purely to authenticate and track 3PID logins and publish end-user public
keys.

> You can host your copy of Sydent, but this will prevent you from reaching other

Reviewer: I think the word `own` was helpful here.

```suggestion
You can host your own copy of Sydent, but this will prevent you from reaching other
```

Author: okk
